### ISSUE TITLE
I cannot replace nodeView with my own view, and have to add VIEW_PART_X, but cannot hid setion 6 content.

### ISSUE THREAD

**Issue by ScottHuangZL** [kartik-v/yii2-tree-manager#23]
**Can we use our own nodeView to replace default view?**

```
'treeViewSettings' => [
                'nodeView' => '@kvtree/views/_form', //Can we replace by our own view??
                'nodeAddlViews' => [
                    kartik\tree\Module::VIEW_PART_1 =>'@app/views/eop/approvaltree/additionalView',
                    kartik\tree\Module::VIEW_PART_2 => '',
                    kartik\tree\Module::VIEW_PART_3 =>'',
                    kartik\tree\Module::VIEW_PART_4 =>'',
                    kartik\tree\Module::VIEW_PART_5 =>'',
                ],
            ],
```
As I know we cannot replaced till April 30,2015.
As alt solution, I have to use VIEW_PART_X to add additional view.
However, I also want to hide your section 6 content. Since it is useless to me due I will use afterFind function to auto setting.

**Can we add a parameters into Module setting to turn on/off session 6?  Just need to add an hidden input for key attributes if turn off that session.**

Thanks.

**Comment by kartik-v** [comment]
I do not understand what you mean by section 6?

**Comment by ScottHuangZL** [comment]
In '@kvtree/views/_form', you have remark for section 1 2 3... 6.
As for section 6, it will show id/name/icon.   Since I actually use my additional VIEW_PART_1 as main view. So would like you help to provide a switch to hide section 6.   Btw, you still need set the id as hidden field since it is key.

Or else, I have to manual edit '@kvtree/views/_form' to hide session 6.

**Comment by kartik-v** [comment]
You should not manually edit the ` @kvtree/views/_form` - because this would get overwritten with future extension updates. You can provide the `nodeView` property to point to another form if you need for advanced use case.

However, for your use case, I would add an enhancement to allow the ID attribute to be hidden if possible.

**Comment by kartik-v** [comment]
Resolved via #25.
